### ISSUE TITLE
add ability to generate individual card images ready for card printing company

### ISSUE THREAD

**Issue by th3tick** [18xx-maker/18xx-maker#458]
The ones I've seen have a card template with essentially three sizes: full size of their template (page size), size of the card (card size), and something about a safety zone where your image must be inside (contents stretched to the edges of to use the space).

The bleed has to get to the full template size, for example, and the actual cut edges of the card itself aren’t that important to the tool itself, even though it seems like it should be.  it’s more like you have to define a template box and a safety box, and what happens in between is not important.  the css padding sometimes also gets in the way so I have to lie about things to get the right size.

**Comment by kelsin** [comment]
I would love to help in this area but from this description don't know what you need. It sounds like individual PDF's or PNG's per card that have options for bleed/border/lines etc?

**Comment by th3tick** [comment]
Yes, that's it exactly.  I've been using boardgamesmaker but they give standard card sizes they and others print.  A few options like mini and mini U.S. versions here would be what I'd suggest to start with:
https://www.boardgamesmaker.com/customized/custom-game-cards.html
